dan_layer/consensus/src/hotstuff/event.rs

```diff
@@ -1,10 +1,12 @@
 //    Copyright 2023 The Tari Project
 //    SPDX-License-Identifier: BSD-3-Clause
 
+use tari_dan_common_types::Epoch;
 use tari_dan_storage::consensus_models::BlockId;
 
 #[derive(Debug, Clone)]
 pub enum HotstuffEvent {
     BlockCommitted { block_id: BlockId },
+    BlockSyncRequest { block_id: BlockId, epoch: Epoch },
```

Comment:
Events are usually named for something (an event) that happened

`BlockchainBehind` or `BlocksOutOfSync` or (insert the best name here)